examples/tutorial/c/tut_allocator.c

```diff
@@ -11,17 +11,21 @@
 #define SIZE 1024
 
 int main() {
+  /* _umpire_tut_c_get_allocator_start */
```

Comment:
Why the change in comment formatting for the C examples?

Reply:
Lol I guess it doesn't matter that much anyways, but hey

Reply:
I was just trying to make it obvious to the viewer that they were seeing C versus C++ versus FORTRAN versus JSON.  We can change it back if it really causes pain.

Reply:
Ah, I see. Again, it's up to you guys, I was just kinda curious. I think it's fine either way!